packages/jupyter-ai/jupyter_ai/magics.py

```diff
@@ -41,6 +44,11 @@ def __init__(self, shell):
         super(AiMagics, self).__init__(shell)
         self.transcript_openai = []
 
+        # suppress warning when using old OpenAIChat provider
+        warnings.filterwarnings("ignore", message="You are trying to use a chat model. This way of initializing it is "
+            "no longer supported. Instead, please use: "
+            "`from langchain.chat_models import ChatOpenAI`")
+
```

Comment:
Does the LLM prediction still work correctly with using `OpenAIChat`?

Reply:
Yes, thanks to the version lock we impose on the `langchain` dependency.